include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -883,7 +953,7 @@ FEPointEvaluation<n_components, dim, spacedim, Number>::evaluate(
                          numbers::signaling_nan<gradient_type>());
     }
 
-  if (unit_points.empty())
+  if (unit_points.size() == 0)
```

Comment:
Let's keep this show it was.

Reply:
I changed this because `ArrayView` does not provide an `empty()` function. If we decide to keep the local vector copy I will revert this change.

Reply:
What about adding `empty()` to `ArrayView`. We can do that in a follow-up PR!